docs/_includes/permissions.html

```diff
@@ -4,63 +4,69 @@
     vertical-align: middle;
     unicode-bidi: isolate;
     border-color: inherit;
-}
+  }
 
-td {
+  td {
     padding: 1rem;
     border-top: 2px solid var(--color-primary-dark);
-}
-
-table tbody tr:nth-of-type(odd) {
-    background-color: var(--color-primary-medium) !important;
   }
 
   @media screen and (min-width: 1366px) {
     table {
-    width: 100%;
-    table-layout: fixed; 
-  }
-  td {
-    word-wrap: break-word; 
+      width: 100%;
+      table-layout: fixed;
+    }
+    td {
+      word-wrap: break-word;
+    }
+    #tablesContainer {
+      max-width: 1200px;
+      margin: auto;
+    }
   }
-  #tablesContainer {
-    max-width: 1200px; 
-    margin: auto;
+
+  .hidden-row {
+    display: none;
   }
-}
 
+  .odd-row {
```

Comment:
Instead of providing odd-row class maybe you can use the nth-child(odd/even) selector.
It should further help in reducing the code.

Reply:
previously that was the approach but odd event does not work in our case because every even row is a hidden row, so by default table will look fully grey

Reply:
Oh I see. makes sense.

Reply:
Maybe instead of even odd you can then try 
![image](https://github.com/meshery/meshery/assets/59254790/6aaf065f-3014-47ab-979e-098367f10d87)

Reply:
this would only select the third item in list, rest of the list will not be selected